pkg/controller/dataimportcron-controller.go

```diff
@@ -138,30 +139,61 @@ func (r *DataImportCronReconciler) initCron(ctx context.Context, dataImportCron
 	return nil
 }
 
-func (r *DataImportCronReconciler) getImageStream(ctx context.Context, imageStreamName, imageStreamNamespace string) (*imagev1.ImageStream, error) {
+func (r *DataImportCronReconciler) getImageStream(ctx context.Context, imageStreamName, imageStreamNamespace string) (*imagev1.ImageStream, string, error) {
 	if imageStreamName == "" || imageStreamNamespace == "" {
-		return nil, errors.Errorf("Missing imagestream name or namespace")
+		return nil, "", errors.Errorf("Missing imagestream name or namespace")
 	}
 	imageStream := &imagev1.ImageStream{}
+	isName, isTag, err := splitImageStreamName(imageStreamName)
```

Comment:
I think the naming is throwing me off here, isName, and isTag I keep thinking why is this not a bool for the 'isSomething' naming convention, but it is actually imageStreamName, and imageStreamTag. maybe not shorten it in this case to make it clearer? Or just name and tag, since we are in the getImageStream function.

Reply:
fixed